### ISSUE TITLE
'Export' object has no attribute 'hostname'

### ISSUE THREAD

**Issue by peerasan** [nicolargo/glances#1899]
**Bug describe** 
Can't send stat to influxdb2

**To Reproduce**
1. glances --export influxdb2 -t 10 -d -q
2. /etc/glances/glances.conf
```
[influxdb2]
host=xxx.xxx.xxx.xx
port=8086
protocol="http"
org="patrickz"
bucket="glances"
token="<INFLUXDB TOKEN>"
```

**Expected behavior**
Glances should send some data to influxdb2

**Server Environment**
 - OS: Ubuntu 20.04 LTS
 - Glances Version: v3.2.1 with PsUtil v5.8.0
 - Influxd:  2.0.7
 - Glances logs file
 ```
2021-07-19 07:33:25,159 -- DEBUG -- Cannot grab IP information: 'Plugin' object has no attribute 'public_address'
2021-07-19 07:33:25,159 -- DEBUG -- Plugin ip update return {'address': 'xxx.xxx.xxx.xxx', 'mask': '255.255.240.0', 'mask_cidr': 20, 'gateway': 'yyy.yyy.yyy.yyy'} in 0.000555 seconds
2021-07-19 07:33:25,159 -- DEBUG -- Stats updated duration: 0.063908 seconds
2021-07-19 07:33:25,159 -- DEBUG -- Export stats using the influxdb2 module
2021-07-19 07:33:25,160 -- DEBUG -- Cannot export cpu stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,160 -- DEBUG -- Stats exported duration: 0.000656 seconds
2021-07-19 07:33:25,160 -- DEBUG -- Cannot export percpu stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,161 -- DEBUG -- Cannot export load stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,161 -- DEBUG -- Cannot export mem stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,161 -- DEBUG -- Cannot export memswap stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,161 -- DEBUG -- Cannot export network stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,161 -- DEBUG -- Cannot export diskio stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,162 -- DEBUG -- Cannot export fs stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,162 -- DEBUG -- Cannot export processcount stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,162 -- DEBUG -- Cannot export ip stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,162 -- DEBUG -- Cannot export system stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,162 -- DEBUG -- Cannot export uptime stats to InfluxDB ('Export' object has no attribute 'hostname')
2021-07-19 07:33:25,162 -- DEBUG -- Cannot export empty sensors stats to InfluxDB
2021-07-19 07:33:25,162 -- DEBUG -- Cannot export empty docker stats to InfluxDB
2021-07-19 07:33:25,162 -- DEBUG -- Cannot export empty gpu stats to InfluxDB
```
 - Glances test [output of glances --issue] (only available with Glances 3.1.7 or higher)
```
glances --issue
=========================================================
Glances version 3.2.1 with PsUtil 5.8.0
=========================================================
alert         [OK]    0.00005s []
amps          [OK]    0.00002s key=name [{'key': 'name', 'name': 'Dropbox', 'result': None, 'refresh': 3.0, 'timer': 0.08350777626037598, 'count': 0, 'countmin': 1.0, 'countmax': None
cloud         [OK]    0.00001s {}
connections   [N/A]
core          [OK]    0.00061s {'phys': 4, 'log': 4}
cpu           [OK]    0.00001s {'total': 26.5, 'user': 18.6, 'nice': 0.0, 'system': 6.8, 'idle': 73.7, 'iowait': 0.0, 'irq': 0.0, 'softirq': 0.8, 'steal': 0.0, 'guest': 0.0, 'guest_ni
diskio        [OK]    0.00004s key=disk_name [{'time_since_update': 1, 'disk_name': 'vda', 'read_count': 0, 'write_count': 0, 'read_bytes': 0, 'write_bytes': 0, 'key': 'disk_name'}, {
docker        [OK]    0.00003s []
folders       [OK]    0.00001s []
fs            [OK]    0.00004s key=mnt_point [{'device_name': '/dev/vda1', 'fs_type': 'ext4', 'mnt_point': '/', 'size': 51848359936, 'used': 32381370368, 'free': 19450212352, 'percent
gpu           [OK]    0.00001s []
help          [OK]    0.00002s None
ip            [OK]    0.00001s {'address': 'xxx.xxx.xxx.xxx', 'mask': '255.255.240.0', 'mask_cidr': 20, 'gateway': 'yyy.yyy.yyy.yyy'}
irq           [N/A]
load          [OK]    0.00003s {'min1': 0.05, 'min5': 0.03, 'min15': 0.0, 'cpucore': 4}
mem           [OK]    0.00001s {'total': 8347992064, 'available': 7676608512, 'percent': 8.0, 'used': 671383552, 'free': 7676608512, 'active': 867848192, 'inactive': 1051619328, 'buff
memswap       [OK]    0.00002s {'total': 2147479552, 'used': 0, 'free': 2147479552, 'percent': 0.0, 'sin': 0, 'sout': 0}
network       [OK]    0.00064s key=interface_name [{'interface_name': 'lo', 'alias': None, 'time_since_update': 0.011920452117919922, 'cumulative_rx': 3021088, 'rx': 0, 'cumulative_tx
now           [OK]    0.00002s 2021-07-19 07:54:02 UTC
percpu        [OK]    0.00001s key=cpu_number [{'key': 'cpu_number', 'cpu_number': 0, 'total': 72.0, 'user': 0.0, 'system': 1.0, 'idle': 28.0, 'nice': 0.0, 'iowait': 0.0, 'irq': 0.0,
ports         [OK]    0.00003s [{'host': 'yyy.yyy.yyy.yyy', 'port': 0, 'description': 'DefaultGateway', 'refresh': 30, 'timeout': 3, 'status': 0.004433, 'rtt_warning': None, 'indice': 'por
processcount  [OK]    0.00003s {'total': 145, 'running': 1, 'sleeping': 90, 'thread': 241, 'pid_max': 0}
processlist   [OK]    0.00006s [{'pid': 200882, 'io_counters': [0, 98304, 0, 0, 0], 'ppid': 2453, 'cpu_percent': 0.0, 'cmdline': ['influxd'], 'cpu_times': pcputimes(user=12.36, system
psutilversion [OK]    0.00001s (5, 8, 0)
quicklook     [OK]    0.00003s {'cpu': 26.5, 'percpu': [{'key': 'cpu_number', 'cpu_number': 0, 'total': 72.0, 'user': 0.0, 'system': 1.0, 'idle': 28.0, 'nice': 0.0, 'iowait': 0.0, 'ir
raid          [N/A]
sensors       [OK]    0.00004s []
smart         [N/A]
system        [OK]    0.00001s {'os_name': 'Linux', 'hostname': 'patrickz-iiwnz', 'platform': '64bit', 'linux_distro': 'Ubuntu 20.04', 'os_version': '5.8.0-59-generic', 'hr_nam
uptime        [OK]    0.00001s {'seconds': 361668}
wifi          [N/A]
```

**Comment by peerasan** [comment]
I quick fix by edit file `./export/glances_influxdb2.py`
Added  `self.hostname = None`

```
        # Mandatories configuration keys (additional to host and port)
        self.org = None
        self.bucket = None
        self.token = None

        # Optionals configuration keys
        self.protocol = 'http'
        self.prefix = None
        self.tags = None
        self.hostname = None #<--------------
```

**Comment by nicolargo** [comment]
Thanks for the patch @peerasan 

Added to the develop branch and will be available in Glances version 3.2.2.
